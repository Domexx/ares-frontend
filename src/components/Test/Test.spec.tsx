import React from 'react';

import { render } from '@testing-library/react';

import Test from './Test';

describe('Test Component', () => {
  it('should render properly', () => {
    const { container } = render(<Test text="Test" />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
