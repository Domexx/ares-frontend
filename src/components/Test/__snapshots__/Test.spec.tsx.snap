// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Component should render properly 1`] = `
<div
  class="test"
>
  <button
    class="bg-red-400 font-bold text-white px-6 py-2"
  >
    Test
  </button>
</div>
`;
